.github/workflows/test-notify-slack.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Run notify-slack action'
+
+on:
+  pull_request:
+    paths:
+      - actions/notify-slack/*
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test-notify-slack-action:
+    name: 'Uses the notify-slack action'
+    runs-on: ubuntu-latest
+    if: always()
+    outputs:
+      status: ${{ steps.notify.outputs.status }}
+    steps:
+      - name: 'Test notify-slack action'
+        id: notify
+        uses: shopsmart/github-actions/actions/notify-slack@feature/slack-notify
+        with:
+          application: github-actions
+          environment: staging
+          status: ${{ needs.jobs.outputs.status }}
+          type: deployment
+          version: v1.0.56
+          template: ./github-actions/actions/notify-slack/message.json.j2
+          slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+      - name: 'Checkout actions'
+        uses: actions/checkout@v3
+
+      - name: 'Check workflow status - failure'
+        id: check-failure
+        if: ${{ steps.notify.outputs.status == 'failure' }}
+        run: echo "Workflow failed"
+
+      - name: 'Check workflow status - cancelled'
+        id: check-cancelled
+        if: ${{ steps.notify.outputs.status == 'cancelled' }}
+        run: echo "Workflow was cancelled"
+
+      - name: 'Check workflow status - success'
+        id: check-success
+        if: ${{ steps.notify.outputs.status == 'success' }}
+        run: echo "Workflow was successful"
```

Comment:
What are these checks for?  These checks are simply going to echo out the workflow status to the action output.

Reply:
Still looking for some clarity on these checks.